cli/tests/mongoDb-psmdb-new.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { test, expect } from '@playwright/test';
+import * as cli from '@helpers/cli-helper';
+
+const version = process.env.PSMDB_VERSION ? `${process.env.PSMDB_VERSION}` : '4.4';
+const shard_container_name = `psmdb_pmm_${version}_sharded`;
+test.describe('Percona Server MongoDB (PSMDB) CLI tests New', async () => {
```

Comment:
Is there a reason to create another test spec file instead of adding the test into existing one? 
What does the "new" word means in spec description? Does this mean that other tests are "old" and needs to be removed? 
Please remove the word "new" and add the proper description which explains what is this test groups for and why we have it separate from existing. Or move test to existing PSMDB group.